vms/platformvm/state/chain_time_helpers.go

```diff
@@ -68,3 +70,10 @@ func GetNextStakerChangeTime(state Chain) (time.Time, error) {
 		return time.Time{}, database.ErrNotFound
 	}
 }
+
+// [PickFeeCalculator] creates either a static or a dynamic fee calculator, depending on the active upgrade
+// [PickFeeCalculator] does not modify [state]
+func PickFeeCalculator(cfg *config.Config, state Chain) (fee.Calculator, error) {
```

Comment:
I assume this takes in the full state because it will be used in the future to access the excess gas values.

Reply:
Correct. Eventually we'll also need parentBlockTimestamp